packages/frontend/src/components/dialogs/ViewProfile/index.tsx

```diff
@@ -220,6 +220,11 @@ export function ViewProfileInner({
     avatarPath = selfChatAvatar
   }
 
+  const mutualChatsMinItems = 5
```

Comment:
Isn't this supposed called `maxItems`? Or rather `maxMinWidthItems`

Reply:
yeah `maxMinWidthItems` would be a more accurate name, thanks

Reply:
*height, sorry XD